collect_app/src/androidTest/java/org/odk/collect/android/DaggerAndroidTest.java

```diff
@@ -0,0 +1,22 @@
+package org.odk.collect.android;
+
+import org.odk.collect.android.application.Collect;
+import org.odk.collect.android.injection.AndroidTestComponent;
+import org.odk.collect.android.injection.DaggerAndroidTestComponent;
+
+import javax.annotation.OverridingMethodsMustInvokeSuper;
+
+public abstract class DaggerAndroidTest {
```

Comment:
Hey @shobhitagarwal1612 do you think we could put a good comment here? I know it's a bit obvious but you know comments really help with bringing across new approaches!

Reply:
Sure